legal-api/src/legal_api/core/filing.py

```diff
@@ -490,10 +492,13 @@ def get_document_list(business, filing, request) -> Optional[dict]:  # pylint: d
             if legal_filings := filing.storage.meta_data.get('legalFilings'):
                 legal_filings_copy = copy.deepcopy(legal_filings)
                 if (filing.filing_type == Filing.FilingTypes.SPECIALRESOLUTION.value and
-                        business.legal_type == Business.LegalTypes.COOP.value and
-                        Filing.FilingTypes.CHANGEOFNAME.value in legal_filings):
-                    # suppress change of name output for MVP since the design is outdated.
-                    legal_filings_copy.remove(Filing.FilingTypes.CHANGEOFNAME.value)
+                        business.legal_type == Business.LegalTypes.COOP.value):
+                    # add special resolution application output
+                    documents['documents']['specialResolutionApplication'] = \
+                        f'{base_url}{doc_url}/specialResolutionApplication'
+                    if Filing.FilingTypes.CHANGEOFNAME.value in legal_filings:
+                        # suppress change of name output for MVP since the design is outdated.
+                        legal_filings_copy.remove(Filing.FilingTypes.CHANGEOFNAME.value)
```

Comment:
I believe we need to remove `alteration` filing type from being returned in the document list as well.

Reply:
Yup that's correct